publications-2016.bib

```diff
@@ -890,26 +810,31 @@ @article{2016_77
    doi     = {}
 }
 
-@article{2016_78,
-   author  = {S. Lee and M. Mikelic and M. F. Wheeler and T. Wick},
-   title   = {Phase-field modeling of proppant-filled fractures in a poroelastic medium},
-   journal = {Comp. Meth. Appl. Mech. Engrg.},
-   year    = {2016},
-   volume  = {312},
-   pages   = {509--451},
-   url     = {http://www.sciencedirect.com/science/article/pii/S0045782516300305},
-   doi     = {10.1016/j.cma.2016.02.008}
+@article{Lee2016,
+  doi = {10.1016/j.cma.2016.02.008},
+  url = {https://doi.org/10.1016/j.cma.2016.02.008},
```

Comment:
what was our rule regarding duplication of doi and url?

Reply:
I'm taking entries verbatim from doi2bib.org, and it has both. I don't think it's reasonable to hand-edit these entries every time -- it's enough work as is :-)

Reply:
I agree that the quality of these entries is not particularly important.